envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/resource/listeners/filters/RBACFilterFactory.kt

```diff
@@ -39,6 +39,9 @@ class RBACFilterFactory(
 
     private val anyPrincipal = Principal.newBuilder().setAny(true).build()
     private val denyForAllPrincipal = Principal.newBuilder().setNotId(anyPrincipal).build()
+    private val allowAllEndpointsForClients = incomingPermissionsProperties.allowedClientsForAllEndpoints.map {
```

Comment:
Isn't it a list of clients? `clientsAllowedToAllEndpoints` or maybe something less mouthful `clientsWithFullAccess` or shorter `fullAccessClients` (alternatively replace `full` with `open` but I think `full` has more emphasis)